canvas_account_admin_tools/settings/local.py

```diff
@@ -64,6 +64,9 @@
          },
       }
    },
+   'cross_listing': {
+      'relative_url': 'accounts/8/external_tools/99'
```

Comment:
Ah, I read and re-read it this comment like 3 times to figure out how this means (and how to fix).  It makes sense now.  I've removed this cross_listing relative URL from local.py.  Thanks.